ReleaseHistory.md

```diff
@@ -14,56 +14,49 @@
 - UEE => eliminate unhandled exceptions in engine
 
 ## **v4.1.0** UNRELEASED
+* BUG: Fix BA2004 HASH not SHA256 message - was hardcoded to SHA-1. [#865](https://github.com/microsoft/binskim/pull/865)
+* DEP: Update Sarif.Sdk submodule from [120fae3 to 2d52c53](https://github.com/microsoft/sarif-sdk/compare/120fae3...2d52c53). Full [SARIF SDK Release History](https://github.com/microsoft/sarif-sdk/blob/2d52c53/src/ReleaseHistory.md).
+* BRK: Remove `--verbose` command-line option (in favor of `--level` and `--kind`). [#853](https://github.com/microsoft/binskim/pull/853)
+* BRK: Remove `--hashes` command-line option (in favor of `--insert Hashes`). [#853](https://github.com/microsoft/binskim/pull/853)
+* BUG: Fix `BA2022.SignSecurely` unhandled `InvalidOperationException`: `Unrecognized crypto HRESULT: 0x80096011`, which is `TRUST_E_MALFORMED_SIGNATURE`, by refreshing `CryptoError` enum with latest data from Windows SDK for Windows 11 (10.0.22621.0). [850](https://github.com/microsoft/binskim/pull/850)
```

Comment:
>BA2022.SignSecurely [](http://example.com/codeflow?start=12&length=19)

this rule reference meets the standard. this is a 'commonly offered review note' or a CORN. Do we have this documented in a CORN document? If not, we should.

All rules need to reference the opaque rule id followed by a dot followed by the rule friendly name.

All 'things' like rule ids are enquoted in markdown backticks.

Definitely need it in our contributors' guidelines.